modin/core/execution/unidist/common/utils.py

```diff
@@ -45,7 +45,8 @@ def initialize_unidist():
             unidist.init()
             """,
         )
-
+        # TODO: allow unidist to inherit env variables on initialization
+        # with set_env(PYTHONWARNINGS="ignore::FutureWarning"):
```

Comment:
@YarShev is it possible to make that change? 

I don't want to force users to set environment variables manually. In addition, in the case of warnings, this environment variable is only needed for worker processes, and not for main one.

Reply:
This change should work when running unidist on MPI with `mpiexec -n <N> python foo.py`. However, this won't have effect when using a dynamic spawn, i.e., `mpiexec -n 1 python foo.py`. Can you create a feature request in unidist repo to add an ability for passing configuration settings in to MPI workers?

Reply:
> This change should work when running unidist on MPI with `mpiexec -n <N> python foo.py`. However, this won't have effect when using a dynamic spawn, i.e., `mpiexec -n 1 python foo.py`. Can you create a feature request in unidist repo to add an ability for passing configuration settings in to MPI workers?

https://github.com/modin-project/unidist/issues/341